packages/api/src/command/medical/document/tsconfig.json

```diff
@@ -3,6 +3,6 @@
   "compilerOptions": {
     "noUncheckedIndexedAccess": true,
   },
-  "include": ["./**/*"],
+  "include": ["./**/*", "../../../../../core/src/external/aws/document-signing/document-bulk-signer-factory.ts", "../../../../../core/src/external/aws/document-signing/document-bulk-signer-lambda.ts", "../../../../../core/src/external/aws/document-signing/document-bulk-signer-local.ts"],
```

Comment:
not sure what happened here. I didnt touch this

Reply:
This happens when we move a file, VSCode tries to keep `tsconfig` referencing the file. We most likely want to remove this.